summary-relationships/src/index-create-repo-relationships.ts

```diff
@@ -0,0 +1,80 @@
+// import { dependencyRepos, repoDependencyMappings, allDepRepos } from "./data";
+import { getClient } from "./get-client";
+import { getDeployDependencies } from "./get-deploy-dependencies";
+import { RelationshipForSync } from "@jupiterone/jupiterone-client-nodejs/dist/types";
+import { waitForJobFinalization } from "./wait-for-job";
+
+require("dotenv").config();
```

Comment:
Can we do `import dotenv from 'dotenv'` and then instantiate the config?

Reply:
👍